packages/@ourworldindata/core-table/src/Transforms.ts

```diff
@@ -311,23 +399,35 @@ const availableTransforms: any = {
 
 export const AvailableTransforms = Object.keys(availableTransforms)
 
+const extractTransformNameAndParams = (
+    transform: string
+): { transformName: string; params: string[] } | undefined => {
+    const words = transform.split(" ")
+    const transformName = words.find(
+        (word) => availableTransforms[word] !== undefined
+    )
+    if (!transformName) {
+        console.warn(`Warning: transform '${transformName}' not found`)
+        return
+    }
+    const params = words.filter((word) => word !== transformName)
+    return { transformName, params }
+}
+
 export const applyTransforms = (
     columnStore: CoreColumnStore,
     defs: CoreColumnDef[]
 ): CoreColumnStore => {
-    defs.forEach((def) => {
-        const words = def.transform!.split(" ")
-        const transformName = words.find(
-            (word) => availableTransforms[word] !== undefined
-        )
-        if (!transformName) {
-            console.warn(`Warning: transform '${transformName}' not found`)
-            return
-        }
-        const params = words.filter((word) => word !== transformName)
-        const fn = availableTransforms[transformName]
+    for (let i = 0; i < defs.length; i++) {
+        const def = defs[i]
```

Comment:
I like to use indexing for-loops only when the index is actually relevant in the loop; otherwise a for-of loop is more straightforward in my mind.
```suggestion
    for (const def of defs) {
```